packages/react/src/components/inlinePrompt/InlinePrompt.tsx

```diff
@@ -30,24 +31,19 @@ export const InlinePrompt: React.FunctionComponent<
     IInlinePromptProps & Partial<ReturnType<typeof mapDispatchToProps>>
 > = ({options, onCancel}) => {
     const choices: JSX.Element[] = _.map(options.userChoice.choices, (choice: string) => (
-        <button
-            type="button"
-            className="btn action mod-danger prompt-action enabled"
-            onClick={options.onClick}
-            key={choice}
-        >
+        <Button className="btn mod-danger prompt-action enabled" onClick={options.onClick} key={choice}>

```

Comment:
Is the `prompt-action` still needed now that we're using the Button component?

```suggestion
        <Button className="mod-danger" onClick={options.onClick} key={choice}>
```

Reply:
For this fix, yes it's still needed.  As the first button relies on the `margin-left` of [.prompt-action](https://github.com/coveo/plasma/pull/2555/files#diff-fd9e26d5638b15caa5356b2d413f8588607d199e3bb25810ab762dcc378c6ff0R44)


With no `.prompt-action`
![image](https://user-images.githubusercontent.com/5728044/158674872-956b1540-ae4f-40bc-a388-8bb1cf1d92e5.png)

Reply:
Lets use an existing class if its only for the margin then, it will avoid having one more css class to maintain.

What about `ml1`?